src/Theming/Fluent9ExplorerTheme.ts

```diff
@@ -0,0 +1,397 @@
+import { Theme } from '@fluentui/react-components';
+
+export const Fluent9ExplorerTheme: Theme = {
+    borderRadiusNone: '0',
```

Comment:
hmm is there a way to extend these from? Seems like we have the same set of variables both in other theme ts files?